airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -2028,12 +2032,14 @@ definitions:
         title: Query Parameters
         description: Specifies the query parameters that should be set on an outgoing HTTP request given the inputs.
         anyOf:
-          - type: string
           - type: object
+            title: Key/Value Pairs
             additionalProperties:
               anyOf:
                 - type: string
-                - $ref": "#/definitions/QueryProperties"
```

Comment:
This is also a regression from this PR: https://github.com/airbytehq/airbyte-python-cdk/pull/452 and after it's corrected in current PR, the `model_to_component_factory` starts failing for multiple places at once. cc @brianjlai to take a look and advice how we can fix this.